### DISCUSSION TITLE
Excel 32 Bit Support Error Msg

### DISCUSSION THREAD

**Discussion by Domaldhino** [DaxStudio/DaxStudio#941] [General]
Hi Darren,
 
I've recently installed Dax Studio 3.0.0 with the default settings. It works with Power BI 64bit but errors with the 32bit Excel. 
 
Could I please get some advise on the issue above? 
Is 32bit not supported any longer as part of the new release? 
 
Appreciate the support,

Dom 
![image](https://user-images.githubusercontent.com/81207136/190277667-3aa31cfa-3662-4b58-abc9-a59351b98e52.png)
![image](https://user-images.githubusercontent.com/81207136/190277689-eac9520b-189f-4dd0-ac9b-e49ad7475ab9.png)

**Comment by dgosbell** [comment]
No this is a bug in the latest updates for Excel. I think this issue was introduced with the August 2022 updates for Excel. I'm working with the Excel team to try and get this fixed. I'm tracking this in issue #884 so I'm going to close this as a duplicate of that. So you can click on the subscribe button if you want to get notifications of updates on that issue. But it looks like it's probably going to require an Excel update to fix this.

**Reply by Domaldhino** [reply]
Thanks Darren, sorry about the duplicate issue, I missed it.

**Reply by Domaldhino** [reply]
I am on on 32 bit Excel 2016

**Reply by dgosbell** [reply]
no worries, Quite a number of people have reported this issue. I'm just trying to link them all back to that original issue so that I don't have to hunt around and close a whole lot of individual issues when we get a fix.
